site/pages/badger-profile/index.js

```diff
@@ -27,7 +27,7 @@ const BadgerProfile = ({ badger }: { badger: Badger }) => {
       <div className={styles.profileContainer}>
         <div className={styles.profilePictureContainer}>
           <img
-            src={badger.imageUrl}
+            src={badger.secondaryImageUrl || badger.primaryImageUrl}
```

Comment:
This is in the internal page, we are not using placeholders for loading speed reasons there since there's only one image to load. Second question is a good one though